lib/Client.js

```diff
@@ -1427,6 +1427,30 @@ class Client extends EventEmitter {
         }).then((guild) => new Guild(guild, this));
     }
 
+    /**
+    * Edit guild channel positions. Note that channel position numbers are lowest on top and highest at the bottom.
```

Comment:
Channel position numbers are a bit more niche than this. They're top to bottom lowest to highest for each type of channel (cat, text, voice). For anyone else confused by this, a diagram I made a little while ago should help

![image](https://media.discordapp.net/attachments/831974405701500931/837279983261057034/unknown.png)

Reply:
I didn’t understand what you want me to change and what should the text be.